data/campaigns/Descent_Into_Darkness/scenarios/10_Alone_at_Last.cfg

```
@@ -588,17 +637,35 @@
 
         [message]
             speaker=Dela Keshar
-            message= _ "I told you, Malin, that I would find you again, and I have kept my word. I will not suffer you to bring any further harm to my people, and so you will not live to see another sunset."
+            message= _ "I finally found you, Malin! On this day, you will pay for what you have done to my people. I will not let you live to see another sunset."
         [/message]
 
         [message]
             speaker=Malin Keshar
-            message= _ "You wrong me, sister. I have only tried to help. Have you not seen fewer orcs this past summer?"
+            message= _ "... I suppose you did vow to kill me..."
         [/message]
 
         [message]
             speaker=Dela Keshar
-            message= _ "I will not listen to any more of your lies. Men, attack!"
+            message= _ "Did you not think I would keep my word? Attack him, men! Do not let the traitor escape!"
+        [/message]
+    [/event]
+
+# in case you do manage to defeat Dela
+    [event]
+        name=last breath
+        [filter]
+            id=Dela Keshar
+        [/filter]
+
+        [message]
+            speaker=Dela Keshar
+            message= _ "Vile fiend... I regret that I was... not strong enough... to destroy you..."
+        [/message]
+
+        [message]
+            speaker=Malin Keshar
+            message= _ "You brought this on yourself..."
```

I think it could be clearer what sort of emotion is being conveyed here. I presume the intent is for Malin to just be confused and not know what he's really thinking, but I think there needs to be either different punctuation or more words to clarify it.
```suggestion
            message= _ "You brought this on yourself... I only ever wanted to protect you."
```
```suggestion
            message= _ "You brought this on yourself!"
```

---

I prefer to leave it slightly vague. Not everything needs to be spelled out to the player crystal clear (imo).